### PR TITLE
improvement(frontend): Update app behavior with invalid tokens

### PR DESCRIPTION
**Pull request by amanape**
**End-user friendly description of the problem this fixes or functionality that this introduces**
If the user has an invalid token from home screen and starts a session, it immediately redirects them back to the home screen because of our invalid token handling. There is no reason to maintain these tokens in the home screen so we clear it there too

- [ ] Include this change in the Release Notes. If checked, you must provide an **end-user friendly** description for your change below

---
**Give a summary of what the PR does, explaining any non-trivial design decisions**
Clears session tokens if in home screen.

---
**Link of any specific issues this addresses**

### CODE REVIEW

**Review by rbren — APPROVED** [review]

**Review by rbren — CHANGES_REQUESTED** [review]
Actually I'm not sure this is the right behavior. Will follow up

**Comment by rbren** [comment]
@amanape here's the desired behavior:

* If there's no session token
  * load /, everything works
  * load /app, redirect to /
* If there's an invalid session token
  * load /, clear token
  * load /app, show an error message and redirect to /
* If there's a valid session token
  * load /, redirect to /app
  * load /app, show the ongoing session


lmk if that's handled here and I'll re-approve

**Comment by amanape** [comment]
Behavior is now:

If token is stored:
Visit `/` -> redirect to `/app`

If invalid token is stored:
Visit `/` -> redirect to `/app` (because token was stored) -> receive invalid token error -> display error toast and redirect to `/`

File: frontend/src/entry.client.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import "./i18n";
 import store from "./store";
 
 async function prepareApp() {
-  if (process.env.NODE_ENV === "development") {
+  if (false && process.env.NODE_ENV === "development") {
```
Comment (rbren) [inline frontend/src/entry.client.tsx:17]: revert?

**Review by rbren — APPROVED** [review]

File: frontend/src/entry.client.tsx (hunk 1 above)
Comment (amanape) [inline frontend/src/entry.client.tsx:17]:
```suggestion
  if (process.env.NODE_ENV === "development") {
```

**Pull request merged**